int32[] free_axes
---
bool success
